disabled/elrondShardCoordinator.go

```diff
@@ -1,35 +1,35 @@
 package disabled
 
-// ElrondShardCoordinator is the disabled shard coordinator instance that satisfies the elrond.sharding.Coordinator interface
-type ElrondShardCoordinator struct {
+// MultiversXShardCoordinator is the disabled shard coordinator instance that satisfies the mx-chain-go.sharding.Coordinator interface
```

Comment:
maybe it can be renamed in `ShardCoordinator`

Reply:
fixed